### PR TITLE
GuzzleFileFetcher inadvertently modifies the fetched JSON

### PR DESCRIPTION
**Pull request by phenaproxima**
By calling `JsonNormalizer::asNormalizedJson()` on the data that it fetches, GuzzleFileFetcher inadvertently removes whitespace from the fetched data -- which technically changes the contents of the string, and thus its SHA256 hash. Oops.

After discussion with @tedbow, there doesn't seem to be any obvious reason why the fetcher would need to normalize the JSON; its job is to fetch some number of bytes from the Internet and return them in an unaltered state. So let's fix this.

### CODE REVIEW

File: tests/Unit/GuzzleFileFetcherTest.php [hunk 1]
```diff
@@ -35,9 +35,17 @@ class GuzzleFileFetcherTest extends TestCase
...
+     *
      * @var string
      */
-    private $testContent = '["oolong","assam","matcha","herbal"]';
+    private $testContent = <<<END
+["oolong",
+  "assam",
+"matcha",
+"herbal"]
+END;
```
Comment (tedbow) [inline tests/Unit/GuzzleFileFetcherTest.php:48]: Could you make the test data an object with keys and the keys be unsorted also? Or have 2 test cases 1 with unsorted keys and 1 with whitespace. 

The previous bug would have showed up in the unsorted keys case also even if there was no whitespace

**Review by tedbow — CHANGES_REQUESTED** [review]

File: tests/Unit/GuzzleFileFetcherTest.php (hunk 1 above)
Comment (phenaproxima) [reply tests/Unit/GuzzleFileFetcherTest.php]: Done.

**Review by tedbow — APPROVED** [review]
Looks good!
